example-config/my.server.name.conf

```diff
@@ -0,0 +1,24 @@
+# nginx configuration for etesync server running on https://my.server.name 
+# typical location of this file would be /etc/nginx/sites-available/my.server.name.conf
+
+server {
+    server_name my.server.name;
+
+    root /path/to/etesync/etesync_server;
+
+    location /static {
+        expires 1y;
+        try_files $uri $uri/ =404;
+    }
+
+    location / {
+        uwsgi_pass      unix:/run/uwsgi/etesync.sock;
+        include         uwsgi_params;
+    }
+
+    # change 443 to say 9443 to run on a non standard port
+    listen 443 ssl; # managed by Certbot
```

Comment:
You probably want:
```                                             
listen 443 ssl;                                                            
listen [::]:443 ssl; 
# Enable these two instead of the two above if your nginx supports http2
# listen 443 ssl http2;                                                            
# listen [::]:443 ssl http2; 
```